framework/src/modules/network/network.js

```diff
@@ -47,35 +50,76 @@ module.exports = class Network {
 
 		this.logger = createLoggerComponent(loggerConfig);
 
+		const storageConfig = await this.channel.invoke(
+			'app:getComponentConfig',
+			'storage'
+		);
+		const dbLogger =
+			storageConfig.logFileName &&
+			storageConfig.logFileName === loggerConfig.logFileName
+				? this.logger
+				: createLoggerComponent(
+						Object.assign({}, loggerConfig, {
+							logFileName: storageConfig.logFileName,
+						})
+				  );
+
+		this.storage = createStorageComponent(storageConfig, dbLogger);
+		this.storage.registerEntity('Peer', Peer);
+
+		try {
+			const status = await this.storage.bootstrap();
+			if (!status) {
+				throw new Error('Cannot bootstrap the storage component');
+			}
+		} catch (err) {
+			throw err;
+		}
+
+		// Load peers from the database that were tried or connected the last time node was running
+		let triedPeers;
+		try {
+			triedPeers = await this.storage.entities.Peer.get();
```

Comment:
If you don't pass `limit` to `get()` it will use the default that is only 10.
I reckon you want to get all peers from the database, if so use `storage.entities.Peer.get({}, { limit: null })`

Reply:
Oh yeah, we want to load all the peers from DB, I will add it in the query 👍